integration-tests/pom.xml

```diff
@@ -22,6 +22,7 @@
     <dependency>
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-junit5</artifactId>
+      <version>3.12.3</version>
```

Comment:
I imagine we want to use the same quarkus version for all packages, so this probably wants to be a maven variable.

Reply:
Update after reading more: actually it seems to be coming from a BOM.  Can this not use that?

Reply:
I think this is part of being compliant with the Quarkus ecosystem. 
[Building my first extension](https://quarkus.io/guides/building-my-first-extension) - Quarkus Platform section
and 
https://quarkus.io/guides/platform